### PR TITLE
chore: enhance bump script to default to a minor version bump

### PR DESCRIPTION
**Pull request by shivlaks**
Also supports any semVer string as input for scenarios where we want to release patches,
major versions, etc.

Made a switch to use `standard-version` to create a commit at the end of the bump.
`standard-version` also takes care of the CHANGELOG generation now.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: bump.sh [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
 /bin/bash ./install.sh
 
-./node_modules/.bin/conventional-changelog -p angular -i CHANGELOG.md -s --context /tmp/context.json
-
 # align "peerDependencies" to actual dependencies after bump
 # this is technically only required for major version bumps, but in the meantime we shall do it in every bump
 /bin/bash scripts/fix-peer-deps.sh
 
+# Generate CHANGELOG and create a commit
+./node_modules/.bin/standard-version --release --skip.tag=true --commit-all
```
Comment (eladb) [inline bump.sh]: Use “npx standard-version” (should have the same effect)

Comment (eladb) [reply bump.sh]: I would actually argue that a tag is not a bad idea. Let’s not skip it.

**Comment by eladb** [comment]
Taking back what I said about the push. Delivlib’s auto-bump will take care of it.

**Comment by eladb** [comment]
So basically just a couple of minor comments in the script.

File: bump.sh (hunk 1 above)
Comment (shivlaks) [reply bump.sh]: > Use “npx standard-version” (should have the same effect)

done

**Comment by shivlaks** [comment]
> Taking back what I said about the push. Delivlib’s auto-bump will take care of it.

I'm guessing you mean the tag. I held off on the branch and push because I felt it was a little heavy-handed. Might be good to test on a couple of releases and make sure we don't observe considerations I might've missed before we add it.

Having said that creating a branch and pushing to it should be as simple as adding it as a `postcommit` as part of the `standard-version` stanza in `package.json`

**Pull request merged**